### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by alessiomazzanti**
docs: change some details in readme

@mgeri puoi farmi la rewiew? :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
 # snd-hello-world
 this is my second hello world repository (this time, created by web interface)
 
-My first hello world has been created from shell, starting from a local folder previously created for that purpose.
+I created my first hello world using manual shell commands, starting from a local folder previously created just for that purpose.
```
Comment (mgeri) [inline README.md:4]: Could be better to use the third person?

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
 # snd-hello-world
 this is my second hello world repository (this time, created by web interface)
```
Comment (mgeri) [inline README.md:2]: Anche qui ehhh

File: README.md (hunk 1 above)
Comment (alessiomazzanti) [reply README.md:4]: Marco, for me it's OK with the first person

File: README.md (hunk 2 above)
Comment (alessiomazzanti) [reply README.md:2]: Anche qui mi piace cosi`

**Pull request merged**